### PR TITLE
Migrate config from index_m to jsonConfig

### PR DESCRIPTION
**Pull request by Jey-Cee**

### CODE REVIEW

**Comment by mcm1957** [comment]
@Jey-Cee 
there are merge conflicts

**Comment by mcm1957** [comment]
@foxriver76 

As registered maintainer (https://github.com/iobroker-community-adapters/info#maintainers) please review / merge and create a new release if OK.

**Comment by foxriver76** [comment]
On mobile it looks okay, I just think the create user button and the info text should be aligned vertically to each other

grafik

grafik

For desktop especially maybe we should put the create user info text already in a new line, currently it looks a bit hard to follow with the eyes imo. 

And as the button for "find bridges" is now gone, could we maybe have an indicator while it is currently searching for the bridge or is it too hard to implement with json config?

**Comment by mcm1957** [comment]
Please note that some (minor in my oppinion) code change is required (and included) too.

**Comment by Jey-Cee** [comment]
> And as the button for "find bridges" is now gone, could we maybe have an indicator while it is currently searching for the bridge or is it too hard to implement with json config?

That was not possible, but i think the autocompleteSendTo should bring the this feature by its self like the sendTo it already has. Therefore i opened an issue: https://github.com/ioBroker/ioBroker.admin/issues/2770

**Comment by Jey-Cee** [comment]
> On mobile it looks okay, I just think the create user button and the info text should be aligned vertically to each other

Without extra CSS styles that looks not very well and we loose space which leads to longer ways to scroll. 
And i think we should avoid using own CSS as this can break in future or on display resolutions we did not tested.

How is it with this?
**sm**
grafik
**md**
grafik
**lg**
grafik
**xl**
grafik

**Comment by Jey-Cee** [comment]
Reminder: After this PR is merged the Text has to be adjusted in weblate.

**Comment by foxriver76** [comment]
> > On mobile it looks okay, I just think the create user button and the info text should be aligned vertically to each other
> 
> Without extra CSS styles that looks not very well and we loose space which leads to longer ways to scroll. And i think we should avoid using own CSS as this can break in future or on display resolutions we did not tested.
> 
> How is it with this? **sm** grafik **md** grafik **lg** grafik **xl** grafik

That looks a lot cleaner to me.

**Comment by mcm1957** [comment]
> Reminder: After this PR is merged the Text has to be adjusted in weblate.

which text has to be adjusted at weblate?
changes at i18n will normally loaded into wwblate anyway.

**Comment by Jey-Cee** [comment]
@mcm1957 I had to revert it while there where merge issues on i18n files. 

    "Turn On with Others": "Einschalten mit anderen states als `on` und `bri`",
    "error creating user: ": "Fehler beim Erzeugen des Benutzers: ",
    "error finding bridge": "Fehler beim Finden der Brücke",
    "ignoreOsram": "Ignoriere Osram",

**Review by foxriver76 — APPROVED** [review]
